docs/_tabsets/install.qmd

```diff
@@ -54,7 +54,7 @@ for installer in installers:
         print()
         print(f"```bash\n{cmd.format(extra=extra)}\n```")
         print()
-        print(f"Connect using [`ibis.{mod}.connect`](../backends/{name.lower()}.qmd#ibis.backends.{mod}.Backend.do_connect).")
+        print(f"Connect using [`ibis.{mod}.connect`](../backends/{name.lower()}#ibis.backends.{mod}.Backend.do_connect).")
```

Comment:
Hm, I'm not sure this is correct. There's no file located at `../backend/duckdb` for example.

Reply:
If this is 404-ing it means the path to the qmd file is wrong. We refer to qmds with `#` in many many places and it appears to be working elsewhere.

Reply:
That said, I *think* this is happening because of the way `include` seems to work: the `..` is interpreted relative to includer, not the included file, so I think the fix is

```suggestion
        print(f"Connect using [`ibis.{mod}.connect`](/backends/{name.lower()}#ibis.backends.{mod}.Backend.do_connect).")
```